lib/Checks/BlockOpeningBraceSpaceBeforeCheck.vala

```diff
@@ -38,7 +38,7 @@ public class ValaLint.Checks.BlockOpeningBraceSpaceBeforeCheck : Check {
         char[] chars = line.to_utf8 ();
         bool mistake_found = false;
         for (int i = 0; i < chars.length; i++) {
-            if (i >= 1 && chars[i] == '{' && chars[i - 1] != ' ' && chars[i - 1] != '(') {
+            if (i >= 1 && !Utils.is_char_inside_string (line, i) && chars[i] == '{' && chars[i - 1] != ' ' && chars[i - 1] != '(') {
```

Comment:
The number of times the function is called (for error free lines) is proportional to n squared, where n is the length of the line.  Is there a more efficient way of doing this?